add change log for 6.4 (#3488)
ROCmSoftwarePlatform/MIOpen: CHANGELOG.md

# Change Log for MIOpen

Full documentation for MIOpen is available [here](https://rocm.docs.amd.com/projects/MIOpen/en/latest/)
## MIOpen 3.4.0 for ROCm 6.4.0
### Added

> * [Conv] enable tuning through miopenSetConvolutionFindMode API 
> * [RNN] Added new algo type (miopenRNNroundedDynamic) for LSTM
> * [TunaNet] enabled NHWC for MI300

Reviewer: ```suggestion
* [Conv] Enabled tuning through the `miopenSetConvolutionFindMode` API 
* [RNN] Added the new algorithm type `miopenRNNroundedDynamic` for LSTM
* [TunaNet] Enabled NHWC for MI300
```

### Optimized

> * updated KernelTuningNet for CK solvers

Reviewer: ```suggestion
* Updated KernelTuningNet for CK solvers
```

### Resolved issues

> - Fixed tuning timing results
> - Accuracy for ASM solvers

Reviewer: ```suggestion
* Fixed tuning timing results
* Accuracy for ASM solvers
```
